src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressbymessagefilter/InputSuppressByMessageFilter.java

```diff
@@ -0,0 +1,26 @@
+/*
+com.puppycrawl.tools.checkstyle.checks.coding.RequireThisCheck
+checkFields = true
+checkMethods = true
+validateOnlyOverlapping = false
+
```

Comment:
Filter config should be defined in comment also.
We already have such filters that migrated to such model of test input

Reply:
@romani  sir, https://github.com/checkstyle/checkstyle/pull/11174#discussion_r781769380
I have stated the error I am getting while configuring filter.

Reply:
Please debug why it happening, unfortunately we need to fix that depending problem